kirega/iReporter-fetchAPI
# Ft user can login #162600141

#### What does this PR do?
Implements login functionality on the ui. As such registered users are able to login
#### Description of Task to be completed?
- Create the login layout
-  Implement fetch API
#### How should this be manually tested?
- Clone the repo
- Open the login.html page in your browser.
- in the username and password section enter your details 
- click the login button
- on successful login, you should be redirected to the dashboard.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12595401/49924888-6c60d600-fec8-11e8-89ad-f264946f665b.png)

#### Questions:
